atom/browser/BUILD.gn

```diff
@@ -310,6 +310,8 @@ source_set("browser") {
       "extensions/api/atom_extensions_api_client.h",
       "extensions/atom_browser_client_extensions_part.cc",
       "extensions/atom_browser_client_extensions_part.h",
+      "extensions/atom_component_extensions.cc",
```

Comment:
since this uses brave_resources.h we need to add a dep for `brave_resources` here too